examples/auth-ssl/index.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Module dependencies.
+ */
+
+var express = require('../..');
+var hash = require('pbkdf2-password')()
+var path = require('path');
+var session = require('express-session');
+var fs = require('fs');
+var path = require('path');
+var https = require('https');
+
+var app = module.exports = express();
+
+// config
+
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, 'views'));
+
+// middleware
+
+app.use(express.urlencoded({ extended: false }))
+app.use(session({
```

Comment:
Is having a session necessary to demonstrate SSL?

Reply:
Perhaps not, but for authentication yes. In this case it is important that software can be configured so that plaintext passwords would not be passed through the net.

Reply:
Thank you for your reply. I'm still not understanding how this necessary to demonstrate SSL. Can you please ellaborate on how having this in the SSL example is useful, as we already have an authentication example: https://github.com/expressjs/express/tree/master/examples/auth

Reply:
OK, I will abandon this one and create a hello world style small example on SSL.